src/commands/build.ts

```diff
@@ -35,7 +36,14 @@ export class BuildCommand implements Command {
     },
     {
       name: 'sources',
+      multiple: true,
       description: 'The sources file to build',
+    },
+    {
+      name: 'sw-precache-config',
+      defaultValue: 'sw-precache-config.js',
+      description: 'Path to an service worker precache configuration to be ' +
```

Comment:
s/service worker/sw-precache (given that this is specific to the tool rather than service workers in general)